### ISSUE TITLE
Accessibility: Narrator does not read button text

### ISSUE THREAD

**Issue by jithinjosepkl** [wixtoolset/issues#6000]
* Which version of WiX are you building with?

3.10.3.3010

* Which version of Visual Studio are you building with (if any)?

VS 2017

* Which version of .NET are you building with?

4.0

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

Windows 10

* Describe the problem and the steps to reproduce it.

Narrator does not read button text. Please see attached video for more details.
[3765694_A11y_MSMPI_MSMPISDKSetupScreen_ScreenReader.zip](https://github.com/wixtoolset/issues/files/3140036/3765694_A11y_MSMPI_MSMPISDKSetupScreen_ScreenReader.zip)

**Comment by barnson** [comment]
Windows Installer is in charge of the UI at runtime. WiX is the build tool and cannot influence runtime behavior except as MSI allows. MSI would need to implement a change like this.
